src/models/table-options.model.ts

```diff
@@ -53,9 +53,6 @@ export class TableOptions {
   // Page offset
   offset: number = 0;
 
-  // Loading indicator
-  loadingIndicator: boolean = false;
```

Comment:
This was not being used so I removed it to stop confusion from people. I can revert this change, if it is meant to be used somewhere.